pkg/operator/app/workflowrundeps.go

```diff
@@ -179,7 +185,9 @@ func NewWorkflowRunDeps(wr *obj.WorkflowRun, issuer authenticate.Issuer, metadat
 
 	wrd := &WorkflowRunDeps{
 		WorkflowRun: wr,
-		Issuer:      issuer,
+		Workflow:    obj.NewWorkflow(client.ObjectKey{Namespace: key.Namespace, Name: wr.Object.Spec.WorkflowRef.Name}),
```

Comment:
It looks like this is in the actual tenant/workflow namespace for now (which is fine). When we implement the new CRD, we'll transition to creating the workflows in the same namespace as the tenant objects (same as webhook triggers etc.), right?

Reply:
Yes, this is on the roadmap for this Sprint. Refactoring the tenant handling is probably what I'll tackle next. There are a few different quirks to it that I did not want to touch yet.